deploy/deployment-manager/deploy.sh

```diff
@@ -40,8 +40,11 @@ ELASTIC_ARTIFACT_SERVER=${ELASTIC_ARTIFACT_SERVER:-https://artifacts.elastic.co/
 DEPLOYMENT_LABELS=${DEPLOYMENT_LABELS:-type=cspm-gcp}
 
 # Set environment variables with the name and number of your project.
-export PROJECT_NAME=$(gcloud config get-value core/project)
-export PROJECT_NUMBER=$(gcloud projects list --filter=${PROJECT_NAME} --format="value(PROJECT_NUMBER)")
+PROJECT_NAME=$(gcloud config get-value core/project)
+export PROJECT_NAME
```

Comment:
I wonder if these need to be exported at all?

Reply:
Maybe not. I don't know if this is being picked up somewhere else. I find a way to test this out